docs/styleguide/PyGuide.md

```diff
@@ -0,0 +1,820 @@
+    when loading. If True, the function will raise an exception if it cannot match the state_dict_to_load
+    parameters to the model's parameters (i.e. if some parameters required by model are missing in
+    state_dict_to_load, or if state_dict_to_load has parameters that could not be matched to model parameters,
+    or if the shape of parameters is not matching). If False, the exception won't be raised.
+    Usually is_resume is specified as False when loading uncompressed model's weights into the model with
+    compression algorithms already applied, and as True when loading a compressed model's weights into the model
+    with compression algorithms applied to evaluate the model.
+    :return: The number of state_dict_to_load entries successfully matched and loaded into model.
+    """
+```
+
+<a id="s3.5.3-classes"></a>
+<a id="353-classes"></a>
+<a id="classes"></a>
+#### 3.5.3 Classes 
+
+Classes should have a docstring below the class definition describing the class.
+
+```python
+class ModelTransformer:
+    """
+    Applies transformations to a Keras model graph.
+    """
+    def __init__(self, model, transformation_layout):
+        """
+        Initializes ModelTransformer 
+
+        :param model: Keras model to be transformed
+        :param transformation_layout: list of transformations
+        """
+        self._model = model
+        self._transformation_layout = transformation_layout
+
+    def transform(self):
+        """
+        Applies transformations to the Keras model.
+
+        :return: transformed Keras model
+        """
+```
+
+<a id="s3.5.4-block-and-inline-comments"></a>
+<a id="354-block-and-inline-comments"></a>
+<a id="block-and-inline-comments"></a>
+#### 3.5.4 Block and Inline Comments 
+
+The final place to have comments is in tricky parts of the code. If you're going to have to explain it 
+in the future, you should comment it now. Complicated operations get a few lines of comments before 
+the operations commence. Non-obvious ones get comments at the end of the line.
+
+```python
+# We use a weighted dictionary search to find out where i is in
+# the array.  We extrapolate position based on the largest num
+# in the array and the array size and then do binary search to
+# get the exact number.
+
+if i & (i-1) == 0:  # True if i is 0 or a power of 2.
+```
+
+To improve legibility, these comments should start at least 2 spaces away from
+the code with the comment character `#`, followed by at least one space before
+the text of the comment itself.
+
+On the other hand, never describe the code. Assume the person reading the code
+knows Python (though not what you're trying to do) better than you do.
+
+```python
+# BAD COMMENT: Now go through the b array and make sure whenever i occurs
+# the next element is i+1
+```
+
+<a id="s3.6-strings"></a>
+<a id="36-strings"></a>
+<a id="strings"></a>
+### 3.6 Strings 
+
+Use 'something' instead of "something"
```

Comment:
Maybe
Use 'something' instead of "something" --> Use `'something'` instead of `"something"`